cluster-autoscaler/core/scaledown/actuation/actuator.go

```diff
@@ -376,3 +385,12 @@ func joinPodNames(pods []*apiv1.Pod) string {
 	}
 	return strings.Join(names, ",")
 }
+
+func SingleRuleDrainConfig(shutdownGracePeriodSeconds int) []kubelet_config.ShutdownGracePeriodByPodPriority {
```

Comment:
nit: IMO this fits much better in priority.go.

Reply:
Moved